### PR TITLE
fix safari image bug

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

fixes: #1266
## Screenshots or screen recordings
rich note paymet portal
browsers with sticker
rich note fix upload image

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1637** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1637" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1637?utm_source=stack-comment).

**Comment by benschac** [comment]
from https://github.com/jbx-protocol/juice-interface/pull/1362

**issue**: this compresses non-square image unnaturally.

We should find another solution here! I had a play but couldn't quite get it. Will leave it in your capable hands 🙏 

| my image | image rendered in payment memo |
| --- | --- |
|  Screen Shot 2022-07-11 at 2 03 56 PM | Screen Shot 2022-07-12 at 6 46 26 AM |

_Originally posted by @aeolianeth in https://github.com/jbx-protocol/juice-interface/pull/1362#discussion_r918365829_

**Comment by benschac** [comment]
> from #1362
> 
> **issue**: this compresses non-square image unnaturally.
> 
> We should find another solution here! I had a play but couldn't quite get it. Will leave it in your capable hands 🙏
> 
> my image	image rendered in payment memo
> Screen Shot 2022-07-11 at 2 03 56 PM	Screen Shot 2022-07-12 at 6 46 26 AM
> _Originally posted by @aeolianeth in [#1362 (comment)](https://github.com/jbx-protocol/juice-interface/pull/1362#discussion_r918365829)_

@aeolianeth I was having trouble testing this locally.

more here: https://discord.com/channels/939317843059679252/939705688563810304/1003667990748860457

From the test and screen shots above this seems to be working. But, I don't want to say 100% if I wasn't able to load a non-square image.

**Comment by tomquirk** [comment]
@benschac still happens for me on Brave

Screen Shot 2022-08-02 at 1 06 06 PM

Using this:

Screen Shot 2022-08-02 at 1 04 35 PM

**Comment by benschac** [comment]
CleanShot 2022-08-05 at 10 00 29@2x

**Comment by benschac** [comment]
The sticker is a little bigger. I could keep stickers exactly the same size with a lil js, But I also figure this page is going to have some more tweaks with the coming UX/UI updates.

File: src/components/RichNote.tsx [hunk 1]
```diff
@@ -47,8 +47,9 @@ export default function RichNote({
                 key={i}
                 src={link}
                 style={{ marginTop: '0.5rem', padding: '0.5rem' }}
+                maxHeight={300}
```
Comment (tomquirk) [inline src/components/RichNote.tsx:50]: **suggestion** I think we can keep this at 6rem

```suggestion
                maxHeight={6rem}
```

**Review by tomquirk — APPROVED** [review]
@benschac hellllll yeah!!!! Thanks so much for persisting with this. Works perfectly. Well done!

File: src/components/RichNote.tsx [hunk 2]
```diff
@@ -47,8 +47,9 @@ export default function RichNote({
                 key={i}
                 src={link}
                 style={{ marginTop: '0.5rem', padding: '0.5rem' }}
+                maxHeight={6rem}
```
Comment (tomquirk) [inline src/components/RichNote.tsx:50]:
```suggestion
                maxHeight="6rem"
```

**Pull request merged**